src/main/scala/is/hail/variant/Genotype.scala

```diff
@@ -78,21 +78,23 @@ abstract class Genotype extends Serializable {
     else
       unboxedPX
 
-  def unboxedDosage: Array[Double] =
+  def unboxedGP: Array[Double] =
     if (unboxedPX == null)
       null
     else if (isDosage)
-      unboxedPX.map(_ * Genotype.dosageNorm)
+      unboxedPX.map(_ * Genotype.gpNorm)
     else
-      Genotype.phredToDosage(unboxedPX)
+      Genotype.plToGP(unboxedPX)
```

Comment:
I disagree.  There is also a normalization here. It talks the PL array to the GP array.